Sources/Polymorphic.swift

```diff
@@ -19,16 +20,11 @@ public protocol Polymorphic {
 
 extension Polymorphic {
     public var float: Float? {
-        guard let double = double else {
-            return nil
-        }
-
-        return Float(double)
+        return double.flatMap { Float($0) }
```

Comment:
@tannernelson I love flat maps for code cov 🙌 

I actually find this easier to read, but I'm willing to budge.